releasetasks/templates/firefox/beetmove_image.yml.tmpl

```diff
@@ -12,6 +12,7 @@
         retries: 5
         routes:
             - tc-treeherder-stage.{{ branch }}.{{ revision_hash }}
+            - tc-treeherder-production.{{ branch }}.{{ revision_hash }}
```

Comment:
Sorry, I didn't verify it earlier. It looks like you don't need to specify `-production`, see https://dxr.mozilla.org/mozilla-central/source/testing/taskcluster/tasks/decision/branch.yml#22

In other words it should look like:
```
- tc-treeherder.{{ branch }}.{{ revision_hash }}
- tc-treeherder-stage.{{ branch }}.{{ revision_hash }}
```